### ISSUE TITLE
The directive name in todoEscape.js is todoBlur

### ISSUE THREAD

**Issue by TemanChan** [COMP3111/questionsJs#16]

**Comment by hunkim** [comment]
Can you add more details?

**Comment by shuchuen** [comment]
In todoEscape.js, the todomvc.directive('todoBlur', function () {...}) should be todomvc.directive('todoEscape', function () {...}), am I right?

**Comment by reynaldiwijaya** [comment]
yes, also found this. It causes an error when we do unit testing

**Comment by evanRoberto** [comment]
it think it is the major bug??

**Comment by hunkim** [comment]
I am not sure since I did not use directive in the questions.html. :-)

**Comment by TemanChan** [comment]
I think it is just a typo. Since the directive is not used in the
questions.html, it will not cause any error. But when we are trying to
write unit tests, the naming conflict will make it impossible to test the
directive in todoEscape.js, which contains 50% of the branches in that
folder. :)

2015-09-21 20:19 GMT+08:00 Sung Kim <notifications@github.com>:
